pgjdbc/src/main/java/org/postgresql/core/Parser.java

```diff
@@ -1024,18 +1026,18 @@ public static JdbcCallParseInfo modifyJdbcCall(String jdbcSql, boolean stdString
 
       int closing = s.indexOf(')', opening);
       for (int j = opening; j < closing; j++) {
-        if (!Character.isWhitespace(sb.charAt(j))) {
+        if (!Character.isWhitespace(sb.charAt(j + prefixLength))) {
```

Comment:
@benbenw , please remove computation `s.indexOf(')', opening)` as well
It can be incorporated into the loop, so the loop could stop either at the first non-whitespace character or at `)`. On top of that, it makes sense to add `j < sb.length()` condition to avoid `AIOBE`